app/javascript/components/widgets/weather/current-weather.test.jsx

```diff
@@ -27,12 +41,147 @@ describe('AnticipatedPrecipitation component', () => {
         precip.push({ precipitation: 0.0 });
       }
     }
-    const weather = { minutely: precip };
+    let byHour =
+      '[\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
+      '      }\n' +
+      '    },\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
+      '      }\n' +
+      '    }]';
+    byHour = JSON.parse(byHour);
```

Comment:
This is repeated a couple times and effectively gets an array of POJOs. Could you make a function that accepts two ids and returns an array of POJOs?

Reply:
+1

Reply:
Done.